examples/tutorials/06_store_encrypted.py

```diff
@@ -0,0 +1,65 @@
+"""
+Encrypt data and store it on the Tangle.
+
+simplecrypt library is needed for this example (`pip install simple-crypt`)!
+"""
+from iota import Iota, TryteString, Tag, ProposedTransaction
+from iota.crypto.addresses import AddressGenerator
+from simplecrypt import encrypt
+from base64 import b64encode
+from getpass import getpass
+
+import json
+
+# Declare an API object
+api = Iota('https://nodes.devnet.iota.org:443', testnet=True)
+
+# Some confidential information
+data = {
+    'name' : 'James Bond',
+    'age' : '32',
+    'job' : 'agent',
+    'address' : 'London',
+}
+
+# Convert to JSON format
+json_data = json.dumps(data)
+
+# Ask user for a password to use for encryption
+password = getpass('Please supply a password for encryption:')
+
+print('Encrypting data...')
+# Encrypt data
+# Note, that in Python 3, encrypt returns 'bytes'
+cipher = encrypt(password, json_data)
+
+# Encode to base64, output contains only ASCII chars
+b64_cipher = b64encode(cipher)
+
+print('Constructing transaction locally...')
+# Convert to trytes
+trytes_encrypted_data = TryteString.from_bytes(b64_cipher)
+
+# Generate an address from your seed to post the transfer to
+my_address = AddressGenerator(b'YOURSEEDFROMTHEPREVIOUSTUTORIAL').get_addresses(
+    start=42,
+)[0]
```

Comment:
❔ Could we provide the seed to `Iota.__init__()` on line 15, and then use `api.get_new_addresses()` here instead?

Reply:
Absolutely! It's less confusing as well, there is no need for user to know about the existence of `AddressGenerator`. :)